.vscode/extensions.json

```diff
@@ -0,0 +1,5 @@
+{
+    "recommendations": [
+        "dbaeumer.vscode-eslint"
+    ]
+}
```

Comment:
not sure whether this could be added. seems env-specific, and iirc we don't do it in other notifier repos

Reply:
agreed - this file can probably go. I had suggested it was useful but not everyone uses vscode!